docs/API.md

```diff
@@ -0,0 +1,26 @@
+# API
```

Comment:
I think a better outline would be:

# APIs

## server.js client-side API

(or whatever the filename is)

## Client socket.io API

(events that the client expects to receive and what happens when the client receives them)

## Server socket.io API

(events that the server expects to receive and what happens when the server receives them)